app/assets/javascripts/webchat.js

```diff
@@ -0,0 +1,112 @@
+(function () {
```

Comment:
Yeah, I like receiving the context as an argument as opposed to on `this` too. I copied this over from a different module in `static` but didn't even notice this.